app/models/measure.rb

```diff
@@ -41,7 +41,18 @@ def relevant_for_country?(country_code)
   end
 
   def excluded_country_list
-    excluded_countries.map(&:description).join(', ').html_safe
+    countries = if exclusions_include_european_union?
+                  # Replace EU members with the EU geographical_area
+                  [GeographicalArea.european_union] + excluded_countries.delete_if(&:eu_member?)
```

Comment:
On a second thought, 
we do not need to actually remove the element from the excluded_countries (`delete_if` or `reject!` ), just filtering it out is enough (`reject`).